api_app/visualizers_manager/classes.py

```diff
@@ -379,3 +379,35 @@ def connector_reports(self) -> QuerySet:
         if configs:
             queryset = queryset.filter(config__in=configs)
         return queryset
+
+
+# IMPORTANT! this function allows to handle the errors in the components render.
+# You must define a function that returns a Visualizable then
+# use this function as a decorator. Ex:
+# @visualizable_error_handler_with_params("field1", VisualizableSize.S_2)
+# def generate_field1(value):
+# ...
+# in case the generation of the field will raise an error
+# this will handle it allowing to render the other components
+def visualizable_error_handler_with_params(
+    error_name: str = "", error_size: VisualizableSize = VisualizableSize.S_AUTO
```

Comment:
I did not find an use case where you actively set the `error_size` parameter.
Imho we can remove this, or did you thought about _other_ cases where this would be usefull?